sdk/Sdk.Generators/ExtensionStartupRunnerGenerator.cs

```diff
@@ -216,7 +216,7 @@ private static string GenerateStartupCodeExecutorClass(IList<string> startupType
                                     {
                                         new global::{{typeName}}().Configure(applicationBuilder);
                                     }
-                                    catch (Exception ex)
+                                    catch (global::System.Exception ex)
```

Comment:
Any other generators need a similar fix?

Reply:
I don't think so. But let me know if you want me to check anything specific.

Reply:
We can keep this PR scoped to `ExtensionStartupRunnerGenerator` and I will prepare followup PRs for other generators.